qutebrowser/mainwindow/prompt.py

```diff
@@ -697,6 +704,11 @@ def _insert_path(self, index, *, clicked=True):
             # On Windows, when we have C:\foo and tab over .., we get C:\
             path = path.rstrip(os.sep)
 
+        if self._expands_user:
+            path = path.replace(os.path.expanduser('~'), '~', 1)
+        else:
+            self._expands_user = False
```

Comment:
This `else:` still seems redundant - if it's already false, no need to set it to false again.